src/EnergyPlus/SingleDuct.cc

```diff
@@ -3411,9 +3430,14 @@ void SingleDuctAirTerminal::SizeSys(EnergyPlusData &state)
                     max(state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).NonAirSysDesHeatVolFlow,
                         this->MaxAirVolFlowRate * this->ZoneTurndownMinAirFrac);
             } else {
-                TermUnitSizing(state.dataSize->CurTermUnitSizingNum).AirVolFlow =
-                    max(state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).NonAirSysDesHeatVolFlow,
-                        this->MaxAirVolFlowRate * this->ZoneMinAirFracDes * this->ZoneTurndownMinAirFrac);
+                if (this->SysType_Num == SysType::SingleDuctVAVReheat && this->DamperHeatingAction == Action::ReverseWithLimits) {
+                    TermUnitSizing(state.dataSize->CurTermUnitSizingNum).AirVolFlow =
+                        max(this->MaxAirVolFlowRateDuringReheat, this->MaxAirVolFlowRate * this->ZoneTurndownMinAirFrac);
+                } else {
```

Comment:
So, what if `MaxAirVolFlowRateDuringReheat > NonAirSysDesHeatVolFlow` as it is for Space4-1 in 5ZoneAirCooled-ReverseWithLimits? Will this reheat coil be oversized?

And why is `NonAirSysDesHeatVolFlow` being used here instead of `DesHeatVolFlow`?

Reply:
Regarding the reheat coil sizing, there have been many adjustments in the past to make sure that the coil is not way oversized. The airflow is one thing, but I think it uses the zone heating load in the sizing calcs (plus an offset for entering air temp). But that's just my recollection without looking at the code. So, maybe what you have a this point is ok, for ReverseWithLimits, anyway.